Airport_Ticket_Booking_System/Services/BookingService.cs

```diff
@@ -0,0 +1,110 @@
+using Airport_Ticket_Booking_System.Models;
+
+namespace Airport_Ticket_Booking_System.Services;
+
+public class BookingService
+{
+    private List<Booking> _bookings = new();
```

Comment:
Having this list and reading from it should be fine in our case.
However, keep in mind that in real scenarios, multiple clients may make requests simultaneously, meaning this in-memory list might not always be up to date.